CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # CHANGELOG
 
+## v0.14.1
+
+*May 2nd, 2022*
+
+This release improves the reliability of the relayer by fixing an edge case where
+some queries would fail when emitted after the block was created but before it was executed.
```

Comment:
```suggestion
This release improves the reliability of the relayer by fixing an edge case where
some queries would fail if they reach a full node after a new block is committed but before the application state updates to reflect the changes in that block.```